src/sysinfo

```diff
@@ -691,6 +691,18 @@ function get_setup()
     fi
 }
 
+function get_recovery_configs()
+{
+    for line in $(kbmadm recovery list -p 2>/dev/null); do
```

Comment:
Initialize `has_recovery` and add a comment explaining its significance outside of this function.  Ugh to the lack fo scoped variables throughout this file.